src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+
+}
+
+.calendar {
+  display: flex;
+  width: #{$calendar-width};
+  flex-wrap: wrap;
+
+  &__fields {
+    @include field-size($field-width);
+    gap: #{$calendar-gap};
+    background-color: #eee;
+    border: #{$border-field} solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.3s;
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index}) {
+        &::before {
+          content: "#{$index}";
+          font-family: Arial, Helvetica, sans-serif;
+          font-size: #{$font-size};
```

Comment:
Inside of loops, put only the styles that are changing, in this case only content should be inside of loop. Move the font properties up.